.sass-lint.yml

```diff
@@ -237,7 +240,7 @@ rules:
 
   # enforce how deeply a selector can be nested (defaults to 2)
   # https://github.com/sasstools/sass-lint/blob/master/docs/rules/nesting-depth.md
-  nesting-depth: 2
+  nesting-depth: 3
```

Comment:
I think 2 is an error and 3 a warning. I just change this from error to warning because we have way too many ocurrence of this to fix it right now. It's a complex task to do.